lib/domain/usecase/send_media_on_web_with_caption_interactor.dart

```diff
@@ -0,0 +1,39 @@
+import 'package:fluffychat/presentation/extensions/send_file_extension.dart';
+import 'package:fluffychat/presentation/extensions/send_file_web_extension.dart';
+import 'package:matrix/matrix.dart';
+
+class SendMediaOnWebWithCaptionInteractor {
+  Future<void> execute({
```

Comment:
Why not `Stream`? What happens if sending the image failed?

Reply:
because `SendMediaOnWebWithCaptionInteractor ` is singleton, so the error will be handled focus at chat screen, I think it's better :)

Reply:
got it. ok